src/content/app/tools/blast/state/epics/tests/blastEpics.test.ts

```diff
@@ -106,16 +104,16 @@ afterAll(() => server.close());
 
 describe('blast epics', () => {
   let store: ReturnType<typeof buildReduxStore>['store'];
-  let shutdownEpics: ReturnType<typeof buildReduxStore>['shutdownEpics'];
+  let shutdownTestEpic: ReturnType<typeof buildReduxStore>['shutdownEpic'];
 
   beforeEach(() => {
-    const { store: _store, shutdownEpics: _shutdownEpics } = buildReduxStore();
+    const { store: _store, shutdownEpic } = buildReduxStore();
```

Comment:
Looks like we can simply do `({ store, shutdownEpic } = buildReduxStore());` to re-assign.